source/LumiCalReco/src/LumiCalClusterer_getCalHits.cpp

```diff
@@ -78,7 +78,9 @@ int LumiCalClustererClass::getCalHits(	EVENT::LCEvent * evt,
 
       } else {
 	arm =(*_mydecoder)( calHitIn )["barrel"]; // from 1 and 2
-	phiCell = (*_mydecoder)( calHitIn )["phi"];
+	//FIXME: add check for phi range?
+	phiCell = (*_mydecoder)( calHitIn )["phi"]; // goes from -phiMax/2 to +phiMax/2
+	if(phiCell < 0 ) phiCell += _cellPhiMax; // need to put into positive range only
```

Comment:
Finally, you explained the problem. Yes, you are right, as for now global->local hit transformation
accounts only for beam_crossing_angle ( that was fast fix when Beata signalized problems
with LumiCal reco ). The problem is now that LumiCal_o1_v01_geo driver makes extra rotation around Z-axis when places second LumiCal barrel - this is in my opinion absolutely
unnecessary ( I suggest to change it ). This is not solving arbitrary rotations of LumiCal 
individual barrels, but this problem we can solve later on if we need this.